collector/hyperv.go

```diff
@@ -309,6 +314,27 @@ func NewHyperVCollector() (Collector, error) {
 
 		//
 
+		HostLPGuestRunTime: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, buildSubsystemName("host_lp"), "guest_run_time"),
```

Comment:
The metric unit will need to be added as a suffix for each of the metric names (E.G. `guest_run_time_percent`).